var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -32,7 +34,8 @@ class PyCython(PythonPackage):
     version('0.23.4',  sha256='fec42fecee35d6cc02887f1eef4e4952c97402ed2800bfe41bbd9ed1a0730d8e')
     version('0.21.2',  sha256='b01af23102143515e6138a4d5e185c2cfa588e0df61c0827de4257bac3393679')
 
-    depends_on('python@2.6:2.8,3.3:', when='@0.23:', type=('build', 'link', 'run'))
+    depends_on('python@2.7:2,3.4:', when='@3.0a1:', type=('build', 'link', 'run'))
```

Comment:
```suggestion
    depends_on('python@2.7:2,3.4:', when='@3:', type=('build', 'link', 'run'))
```
3.0.0 is technically less than 3.0.0a9 due to the way Spack compares versions.